tutorial-701/acquia-pipelines.yaml

```diff
@@ -15,7 +15,8 @@ events:
       # If you have build instructions, put them here. They are not required.
       - build_site:
           script:
-            - echo "Put your build instructions, if any, here."
+            - touch index.html
+            - echo "Pipelines examples 701. You successfully deployed your build to an ODE" > index.html
```

Comment:
If you want this to work, you should remove the touch, and echo to docroot/index.html